sqlglot/dialects/nuodb.py

```diff
@@ -16,12 +42,13 @@ class Tokenizer(tokens.Tokenizer):
             "N'",  # unicodequote
             # ?
         ]
-        COMMENTS = ["--", "//", ("/*", "*/")]
+        COMMENTS = ["--", "//", ("/*", "*/"), ("/* !", "*/;")]
         IDENTIFIERS = ["`", '"']  # ?
         STRING_ESCAPES = ["\\"]
 
         KEYWORDS = {
             **Tokenizer.KEYWORDS,
+            "DATABASE" : TokenType.DATABASE,
```

Comment:
Redundant overwriting, as "DATABASE" Keyword already present in the `tokens.Tokenizer` class